src/ui.form-checkbox-group/index.vue

```diff
@@ -146,6 +146,8 @@ const { labelAttrs, checkboxAttrs, listAttrs } = useAttrs(props);
 provide("setCheckboxGroupCheckedItems", (value) => (checkedItems.value = value));
 provide("checkboxGroupCheckedItems", checkedItems);
 provide("checkboxGroupName", props.name);
+provide("getCheckboxGroupColor", () => props.color);
```

Comment:
get prefix is no needed, usually we use names in provide without it. Please remove it from all places.